guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -458,12 +493,19 @@ Write the ISO to a USB storage device using the *dd* utility or *livecd-tools* i
 
 When you start the physical host and boot from the ISO or the USB storage device, the host connects to {ProjectServer} and starts installing operating system from its kickstart tree.
 
+ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{ProjectName} Tools* repository.
+endif::[]
 
 [[Provisioning_Bare_Metal_Hosts-Implementing_PXE-less_Discovery]]
 === Implementing PXE-less Discovery
 
-{ProjectNameX} provides a PXE-less Discovery service that operates without the need for PXE-based services (DHCP and TFTP). You accomplish this using {ProjectServer}'s Discovery image.
+{ProjectNameX} provides a PXE-less Discovery service that operates without the need for PXE-based services (DHCP and TFTP). You accomplish this using {ProjectServer}'s Discovery image. Once a discovered node is scheduled for installation, it uses `kexec` command to reload Linux kernel with OS installer without rebooting the node.
+
+ifeval::["{build}" == "satellite"]
+[NOTE]
+Discovery `kexec` is a Technology Preview feature.
+endif::[]
```

Comment:
Indeed it is, it's a painful one and we are getting rid of it. We need to be very explicit to maybe stop users from using it before they get into trouble.

Reply:
It's going away anyway in a year or something. I would not be worried too much.